SwiftCompilerSources/Sources/SIL/Instruction.swift

```diff
@@ -824,13 +824,19 @@ class GetAsyncContinuationInst : SingleValueInstruction {}
 final public
 class GetAsyncContinuationAddrInst : SingleValueInstruction, UnaryInstruction {}
 
-
 final public
 class MarkDependenceInst : SingleValueInstruction, ForwardingInstruction {
+  public enum Escaping {
+  case escaping
+  case nonEscaping
+  }
   public var valueOperand: Operand { operands[0] }
   public var baseOperand: Operand { operands[1] }
   public var value: Value { return valueOperand.value }
   public var base: Value { return baseOperand.value }
+  public var escaping: Escaping {
```

Comment:
Why not `public var isNonEscaping: Bool`? I'm not sure what benefit the enum `Escaping` gives us compared to a boolean.
(In C++ we sometimes define such "boolean" enums because C++ doesn't have argument labels. But in swift we have)

It's kind of inconsistent, because we use `Bool` for a lot of other flags.